packages/mui-joy/src/Select/Select.tsx

```diff
@@ -562,8 +564,16 @@ const Select = React.forwardRef(function Select<TValue extends {}>(
       getOptionState,
       listboxRef,
       color,
+      registerHighlightChangeHandler,
+      registerSelectionChangeHandler,
     }),
-    [color, getOptionProps, getOptionState],
+    [
+      color,
+      getOptionProps,
+      getOptionState,
+      registerHighlightChangeHandler,
+      registerSelectionChangeHandler,
```

Comment:
Should we maybe group these four values in an object and give it a more descriptive name, like `selectContextValue`, or something similar that will indicate what's the intention of them? If we decide to add something additional in the future, it will always require changes in userland.

Reply:
Done